core/utils/utils.py

```diff
@@ -276,3 +276,14 @@ def validate_bib_dir():
         return settings.BIB_STORAGE
     else:
         return None
+
+def cache_tag(response, tag_name):
+    """
+    Adds Cache-Tag header or appends to it if it already exists. This is
+    useful for cloudflare so that we can purge based on the tag
+    """
+    if response.has_header('Cache-Tag'):
```

Comment:
Think it's worth documenting that the caller is expected to escape values?

https://support.cloudflare.com/hc/en-us/articles/206596608-How-to-Purge-Cache-Using-Cache-Tags-Enterprise-only-

Reply:
👍